contracts/staking/Staking.sol

```diff
@@ -22,8 +22,9 @@ contract Staking is StakingV2Storage, GraphUpgradeable, IStaking {
     using Stakes for Stakes.Indexer;
     using Rebates for Rebates.Pool;
 
-    // 100% in parts per million
-    uint32 private constant MAX_PPM = 1000000;
+    // For safety purposes we define an activation epoch for using the new rewards
+    // distribution formula
+    uint256 private constant GIP_ACTIVATION_EPOCH = 0; // TODO: change before deployment of implementation
 
     // -- Events --
 
```

Comment:
See a few lines down, `event DelegationParametersUpdated` - let's update the names of the event params `queryFeeCut` and `indexingRewardCut` to the new ones

Reply:
Can we break the subgraph by doing that?

Reply:
It would. we can handle it in the subgraph ahead of time though by changing:
```
    indexer.indexingRewardCut = event.params.indexingRewardCut.toI32()
    indexer.queryFeeCut = event.params.queryFeeCut.toI32()
```

to 
```
    indexer.indexingRewardCut = event.parameters[1] as i32
    indexer.queryFeeCut = event.parameters[2] as i32
    ```

Reply:
we could also use `ethereum.decode()` instead of `parameters` https://github.com/graphprotocol/graph-ts/pull/172/files

Reply:
I think it's a good moment to discuss this as I can change it. We can use this opportunity to make it clearer or just leave it as it is.

The idea of using `queryRewards` instead of `queryFee` is:
- queryRewards: Tokens that comes out of the rebate pool when the allocation is claimed.
- queryFees: Total tokens collected on the allocation.

These two things not necessarily will match, depends on the Cobbs-Douglas formula.